vignettes/custom-steps.Rmd

```diff
@@ -94,14 +94,19 @@ If you are unfamiliar with `R6` classes, here is a short guidance how the argume
 Consider the `step_rcmdcheck()` function ([link to source](https://github.com/ropenscilabs/tic/blob/master/R/steps-rcmdcheck.R)):
 
 ```{r eval = FALSE}
-step_rcmdcheck <- function(warnings_are_errors = TRUE, notes_are_errors = FALSE,
+step_rcmdcheck <- function(...,
```

Comment:
Seems easier to just print the code of the function than to maintain an exact copy here.